### DISCUSSION TITLE
New Discord invite link is https://discord.gg/2QHx9yT5

### DISCUSSION THREAD

**Discussion by raingggg** [raingggg/coctohug#8] [Announcements]
New Discord invite link is: [https://discord.gg/2QHx9yT5](New Discord invite link is https://discord.gg/2QHx9yT5)

Hi All,

We are trying to make our community straightforward and powerful. Please use the new link above. Docs and site link will updated soon. Happy farming!

**Comment by juckbolkyes** [comment]
Nice to let us know this.
